bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/ReadSessionCreator.java

```diff
@@ -103,8 +105,8 @@ public ReadSessionResponse create(
             .getMaxParallelism()
             .orElseGet(
                 () -> {
-                  log.debug("using default parallelism [{}]", config.getDefaultParallelism());
-                  return config.getDefaultParallelism();
+                  log.debug("using default parallelism [{}]", DEFAULT_MAX_PARALLELISM);
+                  return DEFAULT_MAX_PARALLELISM;
```

Comment:
was this previously where we had the division logic of table size / XXX MB?

Reply:
If it is I think we should keep that logic for the time being if we can retrieve a size and default to 10/100K if we can't  once we have a better API we can eliminate the division logic.

Reply:
Yes, it is not new

Reply:
so are we still doing the division some place else?